sdks/java/io/amazon-web-services/src/main/java/org/apache/beam/sdk/io/aws/s3/DefaultS3ClientBuilderFactory.java

```diff
@@ -19,8 +19,8 @@
 
 import com.amazonaws.client.builder.AwsClientBuilder;
 import com.amazonaws.services.s3.AmazonS3ClientBuilder;
-import org.apache.beam.sdk.io.aws.options.S3ClientBuilderFactory;
-import org.apache.beam.sdk.io.aws.options.S3Options;
+import org.apache.beam.sdk.io.aws2.options.S3ClientBuilderFactory;
```

Comment:
Why do we need to change the package name for the classes from the current (v1) `amazon-web-services` module?

Reply:
```
This is needed to avoid name collisions at debug time with the
amazon-web-services module. Also it allows users to migrate
progressively to the new module and even be able to mix both if
required.
```

Reply:
oh wait this is a mess you are right